recipes/elapid/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "elapid" %}
+{% set version = "0.3.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bd1e740b3c7333d9f38720c76ec0fe08bc04156b198337b886e1155902b899c5
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - descartes >=1.1.0
+    - geopandas >=0.7.0
+    - glmnet >=2.0,<2.3  # [not win] windows build unavailable; elapid will use sklearn features instead
```

Comment:
As a suggestion for the next release of elapid, you could consider just issuing a warning if glmnet is not available, remove this dependency and set this recipe to `noarch: python`. The upside is that it will work without any further changes on apple silicon, aarch64, ….

Reply:
I've removed the `glmnet` requirement in the recipe and set it to `noarch: python`. It already has a try/except for importing glmnet so it should be fine as-is.